src/app/aircraft/edit/index.njk

```diff
@@ -61,10 +61,10 @@
       </div>
```

Comment:
Found one typo in file \src\app\verify\registeruser\index.njk:
It says:
You account has now been successfully verified. You can now log on and submit GAR files online.

 Should be:
Your account has now been successfully verified. You can now log on and submit GAR files online.

Reply:
Fixed